test/end2end_test.go

```diff
@@ -317,19 +321,38 @@ func unixDialer(addr string, timeout time.Duration) (net.Conn, error) {
 }
 
 type env struct {
+	name     string
 	network  string // The type of network such as tcp, unix, etc.
 	dialer   func(addr string, timeout time.Duration) (net.Conn, error)
 	security string // The security protocol such as TLS, SSH, etc.
 }
 
+var (
+	tcpEnv        = env{name: "TCP", network: "tcp"}
+	tlsEnv        = env{name: "TLS", network: "tcp", security: "tls"}
+	unixSocketEnv = env{name: "Unix", network: "unix", dialer: unixDialer}
+	allEnv        = []env{tcpEnv, tlsEnv, unixSocketEnv}
```

Comment:
Oh, I missed that you even have "unix"+"tls" in the line noise that was line 329 before. :) I'll restore.